src/mlpack/methods/regularized_svd/regularized_svd_impl.hpp

```diff
@@ -33,9 +33,14 @@ void RegularizedSVD<OptimizerType>::Apply(const arma::mat& data,
                                           arma::mat& u,
                                           arma::mat& v)
 {
+  // batchSize is 1 in our implementation of Regularized SVD.
+  // batchSize other than 1 has not been supported yet.
+  const int batchSize = 1;
```

Comment:
Should we add ``Log::Warn`` here, I think most people will miss the comment. An assertion might be another option, but I I think setting the batch size to one and continue afterwards is just fine.